runtime/autoload/health/nvim.vim

```diff
@@ -144,6 +144,23 @@ function! s:check_performance() abort
           \ ['Install a different Nvim package, or rebuild with `CMAKE_BUILD_TYPE=RelWithDebInfo`.',
           \  s:suggest_faq])
   endif
+
+  " check for slow shell invocation
+  let shell_cmd = 'ls'
```

Comment:
How long `ls` takes and depend on the filesystem and the working directory, and it may also be missing on Windows. I'd suggest `echo`.

Reply:
Great point.
Updated!